src/apps/bm_devkit/bmdk_common/app_main.cpp

```diff
@@ -388,12 +389,13 @@ static void defaultTask(void *parameters) {
   sysConfigurationPartition = &debug_configuration_system;
   NvmPartition debug_cli_partition(debugW25, cli_configuration);
   NvmPartition dfu_partition(debugW25, dfu_configuration);
+  dfu_partition_global = &dfu_partition;
   debugConfigurationInit(&debug_configuration_user, &debug_configuration_hardware,
                          &debug_configuration_system);
   debugNvmCliInit(&debug_cli_partition, &dfu_partition);
   debugPlUartCliInit();
   debugDfuInit(&dfu_partition);
-  bcl_init(&dfu_partition, &debug_configuration_system);
+  bcl_init();
```

Comment:
we no longer need to pass through the dfu partition or sys cfg class through bcl init -> bristlemouth init -> bcmp init -> bcmp dfu/config init!

Reply:
Nice! I appreciate this.